roles/openshift_cli/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
       cli_image: "{{ osm_image | default(None) }}"
       
 - name: Install clients
-  yum: pkg={{ openshift.common.service_type }}-clients state=installed
+  dnf: name={{ openshift.common.service_type }}-clients state=installed
```

Comment:
Thanks @sjenning, if we change this to an `ansible_pkg_mgr` action it should work for either.

```
- name: Install clients
  action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-clients state=present"
  when: not openshift.common.is_containerized | bool
```